.github/workflows/package-cli.yml

```diff
@@ -0,0 +1,74 @@
+name: Package CLI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+      - v8
+      - v7
+    paths-ignore:
+      - 'doc/**'
+      - '.gitpod.yml'
+      - 'README.md'
+  push:
+    branches:
+      - main
+      - v8
+      - v7
+    paths-ignore:
+      - 'doc/**'
+      - '.github/**'
+      - '.gitpod.yml'
+      - 'README.md'
+
+permissions:
+  contents: read
+
+jobs:
+  shared-values:
+    name: Shared Values
+    runs-on: ubuntu-latest
+    outputs:
+      go-version: ${{ steps.set-go-version.outputs.go-version }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - id: set-go-version
+        name: Parse Golang Version
+        run: |
+          go_version=($(grep -E '^go 1\.[[:digit:]]{1,2}' go.mod))
+          echo "golang version: ${go_version[1]}"
+          echo "go-version=${go_version[1]}" >> $GITHUB_OUTPUT
+
+  package-cli:
+    name: Package cli
+    runs-on: ubuntu-latest
+    needs:
+      - shared-values
+    outputs:
+      linux-binaries: ${{ steps.build-linux-binaries.outputs.linux-binaries }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ needs.shared-values.outputs.go-version }}
+
+      - name: Package
```

Comment:
I'm wondering if this is redundant because when `actions/upload-artifact` is pointed to the folder, it archives it into a zip. Indirectly this behavior is described in [this doc](https://github.com/actions/upload-artifact#limitations)

Reply:
Fixed